source/_components/vacuum.template.markdown

```diff
@@ -0,0 +1,110 @@
+---
+layout: page
+title: "Template Vacuum"
+description: "Instructions how to setup Template vacuums within Home Assistant."
+date: 2018-10-07 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Vacuum
+ha_release: 0.80
+ha_iot_class: "Local Push"
+logo: home-assistant.png
+ha_qa_scale: internal
+---
+
+The `template` platform creates vacuums that combine components and provides the
+ability to run scripts or invoke services for each of the start, pause, stop,
+return_to_base, clean_spot, locate and set_fan_speed commands of a vacuum.
+
+To enable Template Vacuums in your installation, add the following to your
+`configuration.yaml` file:
+
+{% raw %}
+```yaml
+# Example configuration.yaml entry
+vacuum:
+  - platform: template
+    vacuums:
+      living_room_vacuum:
+        friendly_name: "Living room vacuum"
+        value_template: "{{ states('input_select.vacuum_state') }}"
+        battery_level_template: "{{ states('input_number.vacuum_battery_level') | int }}"
+        fan_speed_template: "{{ states('input_select.vacuum_fan_speed') }}"
```

Comment:
The standard says that there must be a sample configuration with only required options.

Reply:
But you can put additional examples below.